frontend/src/components/design-system/navigation/NavigationBar.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+import ContainerComponent from '@/components/design-system/containers/ContainerComponent.vue';
+import StackComponent from '@/components/design-system/layout/StackComponent.vue';
+import LogoComponent from '@/components/design-system/style/LogoComponent.vue';
+import BodyText from '@/components/design-system/typography/BodyText.vue';
+
+export interface Props {
+  transparent?: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  transparent: false,
+});
+
+const background = computed(() =>
+  props.transparent ? 'transparent' : undefined
+);
+</script>
+
+<template>
+  <ContainerComponent
+    class="navigation-bar"
+    :background="background"
+    padding="medium"
+  >
+    <div class="grid">
```

Comment:
U této komponenty si úplně nejsem jist, co bych si o ní měl myslet. Pokud se jedná o navigaci, tak by to celý mělo být zabaleno do `<nav>`. Pokud to navigace není, tak by se něměla jmenovat NavigationBar....

Reply:
Viz níže.